### ISSUE TITLE
Evaluating classifier score

### ISSUE THREAD

**Issue by fhennerich** [gfleetwood/crowd-flower-hate-speech#4]
I think you should evaluate the classifiers with test data instead of training data, to prevent overfitting.

**Comment by gfleetwood** [comment]
I mostly agree. What I really should have done is have a separate validation set.
